app/gui/src/dashboard/layouts/Drive/Categories/Category.ts

```diff
@@ -55,6 +62,7 @@ const TRASH_CATEGORY_SCHEMA = z
     type: z.literal('trash'),
     id: z.literal('trash'),
     homeDirectoryId: DIRECTORY_ID_SCHEMA,
+    $$type: z.literal('cloud'),
```

Comment:
@somebody1234 maybe there's a better name for this property though?

Reply:
could go with `backend:` i guess